legislators-social-media.yaml

```diff
@@ -3478,6 +3480,7 @@
     thomas: '01654'
     govtrack: 400074
   social:
+    twitter: lacyclaymo1
```

Comment:
I have, and I have no desire (nor the resources) to make individual decisions to include or exclude based on content. That's why we have a policy.